t/expected_test_results/ingredients/fi-percents.json

```diff
@@ -2,23 +2,23 @@
    "ingredients" : [
       {
          "id" : "en:strawberry",
-         "percent" : 12.3,
-         "percent_estimate" : 12.3,
+         "percent" : "12.3",
+         "percent_estimate" : "12.3",
          "text" : "Mansikka",
          "vegan" : "yes",
          "vegetarian" : "yes"
       },
       {
          "id" : "en:orange",
-         "percent" : 6.5,
-         "percent_estimate" : 6.5,
+         "percent" : "6.5",
+         "percent_estimate" : "6.5",
          "text" : "Appelsiini",
          "vegan" : "yes",
          "vegetarian" : "yes"
       },
       {
          "id" : "en:apple",
-         "percent" : 3.5,
+         "percent" : "3.5",
          "percent_estimate" : 81.2,
```

Comment:
Why percent is string while percent_estimate is a number?

Reply:
It's a long standing issue, not related to this change: scalars in Perl have no type, and internally Perl changes the internal type depending on the last operation made on something. 
Definitely not an ideal situation for languages that expect a specific type in the JSON. In practice we try to force the type on some fields, but I probably didn't do it for the ingredients.